circle.yml

```diff
@@ -1,23 +1,58 @@
 #This file is generated by https://github.com/wcandillon/zorba-circleci-tpl
 machine:
   post:
+    - sudo add-apt-repository -y ppa:fcavalieri/zorba
+    - sudo add-apt-repository -y ppa:fcavalieri/sausalito
```

Comment:
And in zorba server also for the mongodb driver

On Thu, Nov 19, 2015 at 2:41 PM, Federico Cavalieri <
federico.cavalieri@28msec.com> wrote:

> And to create the multithreaded testrunner
>
> On Thu, Nov 19, 2015 at 2:40 PM, Federico Cavalieri <
> federico.cavalieri@28msec.com> wrote:
>
>> Because there are custom packages that need to be installed to make some
>> modules work
>>
>> On Thu, Nov 19, 2015 at 2:31 PM, William Candillon <
>> notifications@github.com> wrote:
>>
>>> In circle.yml
>>> <https://github.com/28msec/zorba/pull/202#discussion_r45339961>:
>>>
>>> > @@ -1,23 +1,58 @@
>>> >  #This file is generated by https://github.com/wcandillon/zorba-circleci-tpl
>>> >  machine:
>>> >    post:
>>> > +    - sudo add-apt-repository -y ppa:fcavalieri/zorba
>>> > +    - sudo add-apt-repository -y ppa:fcavalieri/sausalito
>>>
>>> @fcavalieri <https://github.com/fcavalieri> why do you had these
>>> repositories?
>>>
>>> —
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/28msec/zorba/pull/202/files#r45339961>.
>>>
>>
>>
>